packages/contents/src/emscripten.ts

```diff
@@ -57,6 +57,12 @@ export interface IEmscriptenStream {
   position: number;
 }
 
+export function instanceOfStream(
+  nodeOrStream: IEmscriptenFSNode | IEmscriptenStream,
+): nodeOrStream is IEmscriptenStream {
+  return 'node' in nodeOrStream;
+}
+
 export interface IEmscriptenNodeOps {
   getattr(node: IEmscriptenFSNode): IStats;
```

Comment:
Should this be updated according to the changes above?

```suggestion
  getattr(node: IEmscriptenFSNode | IEmscriptenStream): IStats;
```

Reply:
If so, probably also for the other methods.

Reply:
Ah right I missed it, thanks!